core/cloudflow-spark-testkit/src/main/scala/cloudflow/spark/testkit/TestSparkStreamletContext.scala

```diff
@@ -43,9 +43,8 @@ private[testkit] class TestSparkStreamletContext(override val streamletRef: Stri
                                                  inletTaps: Seq[SparkInletTap[_]],
                                                  outletTaps: Seq[SparkOutletTap[_]],
                                                  override val config: Config = ConfigFactory.empty)
-    extends SparkStreamletContext(StreamletDefinition("appId", "appVersion", streamletRef, "streamletClass", List(), List(), config),
-                                  session) {
-  val ProcessingTimeInterval = 1500.milliseconds
+    extends SparkStreamletContext(StreamletDefinition("appId", streamletRef, "streamletClass", List(), List(), config), session) {
+  val ProcessingTimeInterval = 1.second
```

Comment:
Why was this changed? (I'm fine with it if that works ok, just asking)

Reply:
Will take a look.

Reply:
this should be back to `1500.milliseconds`

Reply:
@maasg should this be configurable? it seems like a 'magic' number.

Reply:
It's a magic number.